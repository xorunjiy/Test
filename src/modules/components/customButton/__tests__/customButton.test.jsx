import React from 'react';
import CustomButton from '../CustomButton';

const props = {
    type: 'button',
    text: 'add task',
    width: 40,
    height: 40,
    fontSize: 24,
    handleEvent: () => {},
    buttonBackground: '#fff',
    buttonBackgroundHover:'#fff',
};

describe('CustomButton snapshot', () => {
    it('should correctly render', () => {

        const component = renderer(<CustomButton {...props}/>);

        expect(component).toMatchSnapshot();
    });
});