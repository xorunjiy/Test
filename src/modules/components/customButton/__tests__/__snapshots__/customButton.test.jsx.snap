// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton snapshot should correctly render 1`] = `
<button
  className=""
  fontSize={24}
  height={40}
  onClick={[Function]}
  type="button"
  width={40}
>
  add task
</button>
`;
